### DISCUSSION TITLE
request for default Grafana

### DISCUSSION THREAD

**Discussion by SiNaPsEr0x** [jheredianet/Teslamate-CustomGrafanaDashboards#13] [Ideas]
Question for the dev, do you think it is possible to set by default that when you open grafana the button appears immediately at the top right?

**Comment by jheredianet** [comment]
Yes, it's possible, but I'd have to make a pull request to the master branch of Teslamate. 
I'll tried before, but sin they are so busy and was waiting a long time, I decided to make this repository as a new one.

**Reply by SiNaPsEr0x** [reply]
In fact I was asking for clarifications on this official teslamate .. you did a really great job and I saw your pull

**Comment by jheredianet** [comment]
Thank you very much. I saw your comment :-)
